### ISSUE TITLE
Save fixed locations

### ISSUE THREAD

**Issue by billbobong** [chatziko/location-guard#80]
I use Location Guard and it would be huuuugely useful if I could save a fixed location. I need more than one fixed location and having to input the fixed location manually every time I change it is a bit of a pain.

**Comment by chatziko** [comment]
Marking as enhancement, I might consider it in a future version.

**Comment by virtual-orbis** [comment]
> I use Location Guard and it would be huuuugely useful if I could save a fixed location. I need more than one fixed location and having to input the fixed location manually every time I change it is a bit of a pain.

I agree, it would be great to have a sort of address book with several locations

**Comment by virtual-orbis** [comment]
I managed to solve this issue creating additional profiles in Chrome browser and installing Location Guard with a different configuration for each one.

**Comment by monster220833** [comment]
goooood

**Comment by eromrab** [comment]
I love this idea, but would also love to add that I'd like to be able to set each website to a specific fixed location. For example, I use YouTubeTV (tv.youtube.com) on my mom's account, but she lives in Buffalo, NY, and I live in Houston, TX.  So it'd be nice to be able to set YouTubeTV to always use the Buffalo, NY, address and yet have normal YouTube set to my local fixed address.
